ipa-core/src/net/server/handlers/query/mod.rs

```diff
@@ -142,8 +150,10 @@ pub mod test_helpers {
             .await;
         let resp = test_server.server.handle_req(req).await;
         let status = resp.status();
-        let body_bytes = hyper::body::to_bytes(resp.into_body()).await.unwrap();
+        let buf = resp.into_body().collect().await.unwrap().aggregate();
```

Comment:
```suggestion
        let buf = resp.into_body().collect().await.unwrap().to_bytes();
```

seems that [to_bytes](https://docs.rs/http-body-util/latest/http_body_util/struct.Collected.html#method.to_bytes) is a better option for your use-case